### PR TITLE
add functionality view profil information

### PR DESCRIPTION
**Pull request by Thomas905**

### CODE REVIEW

**Review by elodiedaubie — CHANGES_REQUESTED** [review]
Hello there,

Responsiveness is great, and the body bugfix is a great help to appreciate your work.
Still, I noticed few things: 

1.  Profile take an "e" at the end of the word in English : route naming and template file should be corrected.
2. **in src/Controller** : as we will develop other features for the user profile (modifications, deletions, uploading profile picture), we should create a ProfileController.
3. **in assets/styles/profil.scss:** 
- class naming might be a problem for future frontend developments because they are too generics (ex : card, btn, center), they should be more specific like "profil-card" for instance.
- It could be a good idea to use "em" instead of "px" for a better responsiveness
- pink circle around the profile picto is missing
4. in templates/profil/index.html.twig : the sentence "tu n'as pas encore de mentor" should be there

My apologies, I could not find the way to comment each file, so I wrote it all in one comment.

Elodie

**Review by elodiedaubie — CHANGES_REQUESTED** [review]
Hi again,

Sorry, I still noticed fews things about your corrections : 

- I see you added "profil" word in classes, but it's in french, the english for it is "profile" with an e
- I think you pushed a wrong file (config/packages/security.yaml), I can't see why it in this PR. This is about access control in another PR and might create a problem in the future.
- In profile controller, method naming should be index instead of "profil"

File: assets/styles/profile.scss [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+@media screen and (min-width: 1024px) {
+    .container-btn-profil{
+        display: flex;
+        justify-content: center;
+        flex-direction: row;
+        align-items: center;
+    }
+    
+}
```
Comment (elodiedaubie) [inline assets/styles/profile.scss:89]: Hello there, "profil" is still in french, it should be in English ("profile") for all the classes names

File: src/Controller/ProfileController.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class ProfileController extends AbstractController
+{
+    /**
+     * @Route("/profile", name="profil")
+     */
+    public function profil(): Response
```
Comment (elodiedaubie) [inline src/Controller/ProfileController.php]: Hello, profile needs an  "e" at the end to be in english

File: templates/components/_header.html.twig [hunk 3]
```diff
@@ -4,6 +4,11 @@
         <p>Le pouvoir de transmettre</p>
     </div>
     <div class="connexion-btn">
-        <a href="/login"><img src="{{ asset('build/images/silhouette-connexion.png') }}" alt="connexion">Mon compte</a>
+        {% if app.user %}
+            <a href="/profil"><img src="{{ asset('build/images/silhouette-connexion.png') }}" alt="connexion">Mon compte</a>
```
Comment (elodiedaubie) [inline templates/components/_header.html.twig]: The link doesn't work anymore ("profile" instead of "profil")

File: assets/styles/profile.scss [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+.card-profil{
+    display: flex;
+    border: none;
+    align-items: center;
+    margin-bottom: 3rem;
+    justify-content: center;
+}
+
```
Comment (elodiedaubie) [inline assets/styles/profile.scss:12]: Hi, the pink circle around the profile picture is missing

**Review by elodiedaubie — CHANGES_REQUESTED** [review]
Hello, I finally learned how to comment files directly, so I put my corrections request there.

**Review by elodiedaubie — DISMISSED** [review]

**Review by elodiedaubie — APPROVED** [review]

**Review by RomainClair — APPROVED** [review]

**Pull request merged**